pkg/crc/machine/bundle/metadata.go

```diff
@@ -237,6 +237,13 @@ func GetBundleNameWithoutExtension(bundleName string) string {
 	return strings.TrimSuffix(bundleName, bundleExtension)
 }
 
+func GetBundleNameWithExtension(bundleName string) string {
```

Comment:
do we need this helper, `-b` currently accepts only the path to the bundle, and that'll always have the extension?

Reply:
@anjannath This is needed because if user already have a bundle extracted in the local directory then  as part of `crc start` we need to identify if the bundle is same what we released with, in subsequent commit you will understand why this helper required.

Reply:
I don't see anything in the subsequent commit *explaining* why it's needed. I've also asked several times to clarify this in the commit log.